src/set_theory/pgame.lean

```diff
@@ -779,12 +800,6 @@ using_well_founded { dec_tac := pgame_wf_tac }
 
 instance : has_sub pgame := ⟨λ x y, x + -y⟩
 
-/-- If `w` has the same moves as `x` and `y` has the same moves as `z`,
-then `w - y` has the same moves as `x - z`. -/
-def relabelling.sub_congr {w x y z : pgame}
-  (h₁ : w.relabelling x) (h₂ : y.relabelling z) : (w - y).relabelling (x - z) :=
-h₁.add_congr h₂.neg_congr
```

Comment:
Any reason for removing this?

Reply:
I had added this lemma in an earlier PR but I end up not using it at all.

Reply:
Mainly because the newer proofs do not use `relabelling` but just `equiv`.

Reply:
I think this one is probably worth having anyway as API - if you're going to have `add` and `neg` then usually you should provide `sub` too.

Reply:
That's what I thought too! I had also added a similar `sub_congr` lemma for the equiv's that I ended up removing.

But I think @semorrison left a comment somewhere about the `sub_congr` lemmas being redundant but I can't seem to find the comment anymore :-/ 

There are no lemmas of the form `sub_...`. I suspect that the intent is to use the `add_comm_group` structures on either `game` or `surreal` and only use `pgame` internally.

Reply:
I did leave a message suggesting removing it, but then deleted it after reconsidering. I think this is a fine to include if you want it.